implentio/ui-toolkit
# Feat: IUT-28 MenuIcon

## Description
Created `MenuIcon` component that takes in icon and onClick props. When hovered, the stroke color changes and when clicked on, the background color changes and a shadow is visible under the icon. If there are alerts, an alert will show on the top right corner of the menu icon.

## Link to JIRA ticket
[Link to JIRA ticket](https://implentio.atlassian.net/browse/IUT-28?atlOrigin=eyJpIjoiOGI2YTFiZDZiZTBmNGY2YTk4MzM2NjJkZTQyZjcwOTUiLCJwIjoiaiJ9)

## Testing Instructions
- Fetch and checkout my branch `git fetch && git checkout feat/IUT-28/MenuIcon`
- `npm run storybook`
- Check to see if stroke color changes when hovering over the menu icon, as well as clicking on it.

## Checklist:
Check what is applicable by putting an 'x' in between the []:
